src/DependencyInjection/Configuration.php

```diff
@@ -57,7 +57,7 @@ public function getConfigTreeBuilder()
                 ->arrayNode('messenger')
                     ->canBeEnabled()
                     ->children()
-                        ->scalarNode('message_bus')->defaultNull()->end()
+                        ->scalarNode('bus')->defaultNull()->end()
```

Comment:
I wonder, what's the point of `defaultNull` if we're setting a default (`messenger.default_bus`) in the extension?

Reply:
I'm not an expert with the configuration, that can probably be replaced by `defaultValue('messenger.default_bus')` here.

Reply:
I just replaced it